soperator/modules/monitoring/variables.tf

```diff
@@ -54,8 +54,8 @@ variable "resources_vm_agent" {
     cpu    = string
   })
   default = {
-    memory = "384Mi"
-    cpu    = "250m"
+    memory = "1Gi"
```

Comment:
Should we up default resources in Soperator helm?

Reply:
we do not have vmagent on soperator helm